Domain Connect Spec Draft.adoc

```diff
@@ -1807,7 +1807,7 @@ See <<example-spf-merge>>.
 
 Some DNS Providers may decide not to support SPFM record. Following alternative solutions shall allow general interoperability of the templates for those providers:
 
-- ignore SPFM - will secure that services would not run into conflict with each other, with a disadvantage of lacking SPF, or
+- onboard the templates with SPFM and follow the guidance at <<SPF record merging, SPF record merging>>, or

```

Comment:
I think if we are in the point about alternatives we don't need this point at all (as this is not an alternative but the main solution)